.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+{
+	"name": "wadm",
+	"image": "mcr.microsoft.com/devcontainers/base:bullseye",
+	"features": {
+		"ghcr.io/devcontainers/features/rust:1": {
+			"version": "latest"
+		}
+	},
+  "containerEnv": {
+		"RUST_LOG": "INFO"
+	},
+	"postCreateCommand": "bash ./.devcontainer/install.sh",
+	"workspaceMount": "source=${localWorkspaceFolder},target=/wadm,type=bind,consistency=cached",
+	"workspaceFolder": "/wadm"
+}
```

Comment:
If there are any extensions or settings you want to use, or to use a non-root user you can include as: 
	"customizations": {
		"vscode": {
			"settings": {
				"lldb.executable": "/usr/bin/lldb",
				// Tell VSCode not to watch target dir
				"files.watcherExclude": {
					"**/target/**": true
				},
				"rust-analyzer.checkOnSave.command": "clippy",
				"[rust]": {
					"editor.formatOnSave": true
				}
			},
			"extensions": [
				"vadimcn.vscode-lldb",
				"mutantdino.resourcemonitor",
				"rust-lang.rust-analyzer",
				"tamasfe.even-better-toml",
				"serayuzgur.crates"
			]
		}
	},
	// Use 'forwardPorts' to make a list of ports inside the container available locally.
	// "forwardPorts": [],
	// Use 'postCreateCommand' to run commands after the container is created.
	// "postCreateCommand": "",
	// Set `remoteUser` to `root` to connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
	"remoteUser": "vscode"